### ISSUE TITLE
[08\05_stylenet.py] cannot show the correct picture.

### ISSUE THREAD

**Issue by yingwei13mei** [nfmcclure/tensorflow_cookbook#18]
When I run "tensorflow_cookbook-master\08_Convolutional_Neural_Networks\05_Stylenet_NeuralStyle\05_stylenet.py" , it comes out the below message:
D:\Anaconda3\lib\site-packages\scipy\misc\pilutil.py:98: RuntimeWarning: invalid value encountered in greater
  bytedata[bytedata > high] = high
D:\Anaconda3\lib\site-packages\scipy\misc\pilutil.py:99: RuntimeWarning: invalid value encountered in less
  bytedata[bytedata < 0] = 0
Generation 1000 out of 10000
Generation 1500 out of 10000
Generation 2000 out of 10000
Generation 2500 out of 10000
...
And finally, I got all the black picture. Are the two warning caused the black picture?

**Comment by nfmcclure** [comment]
I investigated this Monday.  It appears the 'style_loss' gets larger and larger each run, eventually hitting `Inf` values, which makes the image go black.

I've played around with the convergence parameters (learning rates and image weights), but no avail.  This is a deeper bug in the loss function somewhere.  This requires more investigation and I suspect the auto-upgrade to TensorFlow version 1.0.0 changed how some mathematical function works.

For now, I'll have to put this off until I can test this in more detail.

If you need to get something working right now, I suggest trying out other versions of this algorithm on github here:
https://github.com/machrisaa/stylenet
https://github.com/cysmith/neural-style-tf

I'll update here in a few weeks when I get this working.  Thanks for bringing this to my attention!

**Comment by nfmcclure** [comment]
This is fixed, thanks to this pull request:
https://github.com/nfmcclure/tensorflow_cookbook/pull/73

Try recloning/forking and run again. Thanks.
